src/playground-helper.ts

```diff
@@ -0,0 +1,26 @@
+import {compressToEncodedURIComponent} from 'lz-string'
+
+function unindent(value: string) {
+  // remove white spaces first, to save a few bytes.
+  // testing-playground will reformat on load any ways.
+  return value.replace(/[ \t]*[\n][ \t]*/g, '\n')
+}
+
+function encode(value: string) {
+  return compressToEncodedURIComponent(unindent(value))
+}
+
+function getPlaygroundUrl(element: Element) {
+  // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+  if (!element) {
+    throw new Error(`The element you're providing isn't a valid DOM element.`)
```

Comment:
There's an existing test in `screen.js`, testing that you can call `logTestingPlaygroundURL` with `null`: https://github.com/testing-library/dom-testing-library/blob/c273ed518cf783591692be8c0c5e5a34feafd4bb/src/__tests__/screen.js#L44

That test should also be removed if we remove this check. Otherwise we should expand the type for both `getPlaygroundUrl` and `logTestingPlaygroundURL` to use `Element | null`.

I'm leaning towards removing the check and the test. Not sure what you prefer?

Reply:
> I'm leaning towards removing the check and the test.

That would be a breaking change. Let's not change unrelated code in this PR and match the types.

Reply:
The test has been re-added 👍